src/frontend/static/images/credits.txt

```diff
@@ -0,0 +1,2 @@
+bicycle-on-red-wall-narrow.jpg,,https://unsplash.com/photos/HALe2SmkWAI
+bicycle-on-red-wall-wide.jpg,,https://unsplash.com/photos/HALe2SmkWAI
```

Comment:
I've created a new credits.txt for the two new images we got from unsplash.com — copying the format of the [existing credits.txt file](https://github.com/GoogleCloudPlatform/microservices-demo/blob/release/v0.2.4/src/frontend/static/img/products/credits.txt).